### DISCUSSION TITLE
研究：D1 晶片效能達人腦 36%，特斯拉晶片「智力」2023 年超越人類

### DISCUSSION THREAD

**Discussion by neil0258** [1101-GS3073-C/Python-Community-GS3073-C#630] [General]
英國汽車租賃公司 Vanarama 最新研究顯示，深入分析過去和現在特斯拉（Tesla）AI 微晶片發現，效能以每年驚人的 486% 速率增長，最新 D1 微晶片每秒運算次數可達 362 兆次，人類大腦每秒 1 千兆次。換言之，D1 微晶片處理效能達大腦 36%，報告預測，2023 年特斯拉微晶片「智力」將超越人類。
即使人類大腦智力也是逐步成長，嬰兒大腦容量出生後一年增加三倍，一直到 25 歲，大腦才完全發展成熟（含 1 千億個神經元）。Vanarama 就是看到特斯拉微晶片效能驚人的成長率，大膽預測從開發時效能一路達到成熟人類大腦的智力水準，只需 17 年就能搞定。

我的想法：研究人員早就說過，人工智慧「總有一天」會超越人類，如何應對比人類更聰明的AI是我們這世代所必須面對的問題，要與AI和平共處，互相幫助，究竟有沒有可能...我認為十分不容易。

**Comment by awkward-willy** [comment]
很難想像當人類已不再是地球上最聰明的存在，更何況還是被自己所創造出的晶片所超越，但也不難看出隨著科技不斷演變，我們若只是一昧的不求精進，終將被科技所吞噬。我們應該更多的去發展創新且獨立思考、以及解決問題的能力。

**Comment by 0212Jerry** [comment]
超過是可以超過，但就難在晶片的製造，如何能在晶片內塞入更多的元件，亦或是能有新的方法，讓整體運算速度大幅提升，拉到一個新的檔次都需要時間來驗證。

**Comment by Vincent20030314** [comment]
看來以後有錢人都會變成聰明人了。
